Source/Core/Core/PowerPC/PPCAnalyst.h

```diff
@@ -154,6 +154,9 @@ struct CodeBlock
 
 	// Did we have a memory_exception?
 	bool m_memory_exception;
+
+	// Which GQRs this block uses, if any.
```

Comment:
I think I commented on this before... but this description doesn't completely match PPCAnalyst in the gqrModified case.  Maybe a comment is enough... or you could add an explicit gqrModified check to where we set assumeNoPairedQuantize?